scriptmodules/supplementary/moonlight-embedded.sh

```diff
@@ -0,0 +1,139 @@
+    make
+    popd
+}
+
+function install_moonlight-embedded() {
+    # Install moonlight-embedded simple theme
+    mkdir -p /etc/emulationstation/themes/simple/moonlight-embedded/art
+    cp -v "$md_build/installer/themes/simple/theme.xml" /etc/emulationstation/themes/simple/moonlight-embedded/theme.xml
+    cp -v "$md_build/installer/themes/simple/art/icon.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/icon.png
+    cp -v "$md_build/installer/themes/simple/art/logo.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/logo.png
+    cp -v "$md_build/installer/themes/simple/art/background.png" /etc/emulationstation/themes/simple/moonlight-embedded/art/background.png
+
+    # Install moonlight-embedded carbon theme
+    mkdir -p /etc/emulationstation/themes/carbon/moonlight-embedded/art
+    cp -v "$md_build/installer/themes/carbon/theme.xml" /etc/emulationstation/themes/carbon/moonlight-embedded/theme.xml
+    cp -v "$md_build/installer/themes/carbon/art/system.svg" /etc/emulationstation/themes/carbon/moonlight-embedded/art/system.svg
+    cp -v "$md_build/installer/themes/carbon/art/controller.svg" /etc/emulationstation/themes/carbon/moonlight-embedded/art/controller.svg
+
+    md_ret_files=(
+        'moonlight/build/moonlight'
+        'moonlight/build/libmoonlight-pi.so'
+        'moonlight/build/libgamestream/libgamestream.so'
+        'moonlight/build/libgamestream/libgamestream.so.0'
+        'moonlight/build/libgamestream/libgamestream.so.2.2.2'
+        'moonlight/build/libgamestream/libmoonlight-common.so'
+        'moonlight/build/libgamestream/libmoonlight-common.so.0'
+        'moonlight/build/libgamestream/libmoonlight-common.so.2.2.2'
+        'moonlight/build/docs'
+    )
+}
+
+function configure_moonlight-embedded() {
+    # Create romdir moonlight embedded
+    mkRomDir "moonlight-embedded"
+
+    # Add System to es_system.cfg
+    addSystem 1 "$md_id" "moonlight-embedded" "%ROM%" "$rp_module_desc" ".sh .SH"
+
+    [[ "$md_mode" == "remove" ]] && return
+
+    # Create moonlight embedded config script
+    cat > "$romdir/moonlight-embedded/moonlight-embedded_config.sh" << _EOF_
+#!/bin/bash
+sudo $scriptdir/retropie_packages.sh moonlight-embedded configure
+_EOF_
+
+    gui_moonlight-embedded
+
+    # Remove existing scripts if they exist & Create scripts for running moonlight-embedded from emulation station
+    cat > "$romdir/moonlight-embedded/moonlight-embedded_720p60fps.sh" << _EOF_
+#!/bin/bash
+pushd "$md_inst"
+./moonlight stream -720 -60fps -app Steam
+popd
+_EOF_
+
+    cat > "$romdir/moonlight-embedded/moonlight-embedded_1080p30fps.sh" << _EOF_
+#!/bin/bash
+pushd "$md_inst"
+./moonlight stream -1080 -30fps -app Steam
+popd
+_EOF_
+
+    cat > "$romdir/moonlight-embedded/moonlight-embedded_1080p60fps.sh" << _EOF_
+#!/bin/bash
+pushd "$md_inst"
+./moonlight stream -1080 -60fps -app Steam
+popd
+_EOF_
+
+    # Chmod scripts to be runnable
+    chmod +x "$romdir/moonlight-embedded/"*.sh
+}

```

Comment:
I would prefer this added to ports rather than own system, with a single launch script - but can have multiple commandlines which are switchable from the runcommand menu. The old limelight module is similar, but it too needs fixing up really (or mostly likely removing).